cypress/integration/details-test.js

```diff
@@ -0,0 +1,52 @@
+describe('', () => {
```

Comment:
Maybe we can name it something like 'Modal' or 'User can observe an accurate modal'

Reply:
Thank you for catching this oversight. Your suggestion will be written in the next branch.